### PR TITLE
FoundationSync: Add endpoint to resolve room names

### PR DESCRIPTION
**Pull request by pferreir**
To be used by Gesloc to redirect users to Indico.

### CODE REVIEW

File: foundationsync/indico_foundationsync/blueprint.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# This file is part of Indico.
```
Comment (ThiefMaster) [inline foundationsync/indico_foundationsync/blueprint.py]:
```python
# This file is part of the CERN Indico plugins.
# Copyright (C) 2014 - 2019 CERN
#
# The CERN Indico plugins are free software; you can redistribute
# them and/or modify them under the terms of the MIT License; see
# the LICENSE file for more details.
```

File: foundationsync/indico_foundationsync/blueprint.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+@blueprint.route('!/rooms/resolve/<building>/<floor>-<number>')
+def resolve_room(building, floor, number):
+    room = Room.query.filter(
+        ~Room.is_deleted,
+        Room.building == building,
+        Room.floor == floor,
+        Room.number == number
+    ).one_or_none()
+    if not room:
+        raise NotFound
```
Comment (ThiefMaster) [inline foundationsync/indico_foundationsync/blueprint.py]: There's also `first_or_404()` (not sure if there's `one_or_404()` as well)

Comment (pferreir) [reply foundationsync/indico_foundationsync/blueprint.py]: Nope. And I want it to error if there's more than one result.

File: foundationsync/indico_foundationsync/blueprint.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+from __future__ import unicode_literals
+
+from flask import redirect
+from werkzeug.exceptions import NotFound
+
+from indico.core.plugins import IndicoPluginBlueprint
+from indico.modules.rb.models.rooms import Room
+
+
+blueprint = IndicoPluginBlueprint('foundationsync', 'indico_foundationsync')
```
Comment (ThiefMaster) [inline foundationsync/indico_foundationsync/blueprint.py]: `__name__` for the second arg? at least we do that in pretty much all other plugins..

**Pull request merged**